latex/content/tables.tex

```diff
@@ -112,31 +119,31 @@ \section{Tabellen}
       \centering
       \caption{Eine schöne Tabelle mit Messdaten.}
       \sisetup{table-format=1.2}
-      \begin{tabular}{S[table-format=3.0] S S S S[table-format=3.2]}
+      \begin{tblr}{colspec={S[table-format=3.0] S S S S[table-format=3.2]}}
         \toprule
-        {$f$} & {$l_\text{start}$} & {$l_1$} & {$l_{\text{kor},1}$} & {$B_1$} \\
+        {{{$f$}}} & {{{$l_\text{start}$}}} & {{{$l_1$}}} & {{{$l_{\text{kor},1}$}}} & {{{$B_1$}}} \\
```

Comment:
woa.. geht das nicht geiler? z.B. damit nen rowspec für Zeile 1 zu setzen oder so?

Reply:
We could of course use the `guard` keyword for row 1 instead. Something like

```
\begin{tblr}{
    colspec = {...},
    row{1} = {guard}
  }
...
\end{tblr}
```
should work well.

Reply:
and you can also add `$` to make them math mode by default

Reply:
Yes, that thought crossed my mind, too. We should mention, however, that one has to be careful when mixing text and math modes in the column names.

Reply:
I would argue for `mode=math` instead of `$`, as the latter messes up the syntax highlighting in minted.